src/app/game/board/board.service.js

```diff
@@ -0,0 +1,17 @@
+import * as dealer from '../../../logic/dealer/dealer';
+import {GameState} from "../../../logic/state";
+
+export default class BoardService {
+
+    static moveCard(card, sourcePile) {
+        dealer.handleMoveCard(card, sourcePile);
+
+        // TODO: remove this from here. its temporary just to see that the moves are working
```

Comment:
I think you forgot to remove this

Reply:
we still need it. when we'll start the game flow we'll delete it.